pkg/controller/controller.go

```diff
@@ -30,10 +34,29 @@ type ControllerConfig struct {
 	DomainWhitelist []string
 }
 
+var wsconn net.Conn
+
 func New(config ControllerConfig) Controller {
 	return Controller{Network: "tcp", Connection: make(chan net.Conn, 1024), ControllerConfig: config, startchan: make(chan error), SelfCertCache: "ligolo-selfcerts"}
 }
 
+type myHttpServer struct {
```

Comment:
Rename struct to more compliant name, like ligoloHttpServer

Reply:
Done in last commit